addons/binding/org.openhab.binding.pentair/ESH-INF/thing/easytouch.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="pentair"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+        
+    <thing-type id="easytouch">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="ip_bridge" />
+            <bridge-type-ref id="serial_bridge" />
+        </supported-bridge-type-refs>
+        
+        <label>EasyTouch Controller</label>
+        <description>Pentair EasyTouch Controller</description>
+        
+        <channels>
+            <channel id="pooltemp" typeId="temperature"/>
+            <channel id="spatemp" typeId="temperature"/>
+            <channel id="airtemp" typeId="temperature"/>
+            <channel id="solartemp" typeId="temperature"/>
+            
+            <channel id="spaheatmode" typeId="heatmode"/>
+            <channel id="poolheatmode" typeId="heatmode"/>
+            <channel id="spaheatmodestr" typeId="heatmodestr"/>
```

Comment:
These heatmodestr channels do not seem to make much sense - are they in any way different to heatmode?